powertools-idempotency/src/main/java/software/amazon/lambda/powertools/idempotency/IdempotencyConfig.java

```diff
@@ -13,8 +13,10 @@
  */
 package software.amazon.lambda.powertools.idempotency;
 
+import com.amazonaws.services.lambda.runtime.Context;
 import software.amazon.lambda.powertools.idempotency.internal.cache.LRUCache;
 
+import java.security.MessageDigest;
```

Comment:
Is this only being pulled in because of the javadoc?

Reply:
looks like yes. I remove this.